src/main/java/com/faforever/client/player/FriendJoinedGameNotifier.java

```diff
@@ -30,15 +30,19 @@
   private final JoinGameHelper joinGameHelper;
   private final PreferencesService preferencesService;
   private final AudioService audioService;
+  private List<FriendJoinedGameEvent> eventList;
```

Comment:
Since it's initialized in the constructor and not meant to be changed, this should be `final`.